extensions/interactions/rule_templates.json

```diff
@@ -236,5 +236,21 @@
     "FuzzyEquals": {
       "description": "is equal to {{x|NormalizedString}}, misspelled by at most one character"
     }
+  },
+  "RulesHtmlMapping": {
+    "ListOfSetsOfHtmlStrings": [
+      ["DragAndDropSortInput", "IsEqualToOrdering", "x"],
```

Comment:
On reflection, I suggest that you make these dicts instead with keys interaction_id, rule_name, input_variable. Otherwise it's not very readable (you have to do x[1]). Sorry for not realizing this before.

Make sure that your backend tests rigorously check the format of this file and all subparts of it.

Reply:
done